packages/connect-wallet-modal/src/Terms/Terms.tsx

```diff
@@ -1,22 +1,34 @@
-import { FC } from 'react';
+import { ChangeEvent, FC } from 'react';
 import { Checkbox, CheckboxProps, Link } from '@lidofinance/lido-ui';
 import { TermsStyle, TermsTextStyle } from './styles';
+import { Metrics } from '../WalletsModal';
 
 type WalletModalConnectTermsProps = Pick<
   CheckboxProps,
-  'checked' | 'onChange' | 'children'
->;
+  'checked' | 'onChange'
+> & { metrics?: Metrics };
 
-export const Terms: FC<WalletModalConnectTermsProps> = (props) => (
-  <TermsStyle>
-    <Checkbox {...props} />
-    <TermsTextStyle>
-      I have read and accept{' '}
-      <Link href="https://lido.fi/terms-of-use">
-        Terms&nbsp;of&nbsp;Service
-      </Link>{' '}
-      and <Link href="https://lido.fi/privacy-notice">Privacy&nbsp;Notice</Link>
-      .
-    </TermsTextStyle>
-  </TermsStyle>
-);
+export const Terms: FC<WalletModalConnectTermsProps> = (props) => {
+  const { onChange, metrics } = props;
+  const onClickTermsAccept =
+    metrics?.events?.click?.handlers.onClickTermsAccept;
+
+  const handleCheckboxChange = (e: ChangeEvent<HTMLInputElement>) => {
+    onChange?.(e);
+    onClickTermsAccept?.();
```

Comment:
Changing checkbox state to unchecked will also trigger onClickTermAccept, is this inteded?

Reply:
It was intended, but now I think that it is wrong. Will fix, thank you

Reply:
@kozlovzxc-at-lido fixed